samples/kvsWebRTCClientMasterGstSample.c

```diff
@@ -3,6 +3,8 @@
 #include <gst/app/gstappsink.h>
 
 extern PSampleConfiguration gSampleConfiguration;
+RTC_CODEC audioCodec = DEFAULT_AUDIO_TRACK_ID;
```

Comment:
Why do we need these if the values in PSampleConfiguration are being set and we can use those in `on_new_sample` callback?

Reply:
Done